DQM/SiTrackerPhase2/python/Phase2TrackerMonitorDigi_cfi.py

```diff
@@ -2,136 +2,137 @@
+                            xmin = cms.double(-0.5),
+                            xmax = cms.double(99.5),
+                            switch = cms.bool(True)
+                        ),
+                        DigiOccupancySH = cms.PSet(
+                            Nbins = cms.int32(51),
+                            xmin = cms.double(-0.001),
+                            xmax = cms.double(0.05),
+                            switch = cms.bool(True)
+                        ),
+                        DigiOccupancyPH = cms.PSet(
+                            Nbins = cms.int32(51),
+                            xmin = cms.double(-0.0001),
+                            xmax = cms.double(0.005),
+                            switch = cms.bool(True)
+                        ),
+                        ChargeXYMapH = cms.PSet(
+                            Nxbins = cms.int32(450),
+                            xmin   = cms.double(0.5),
+                            xmax   = cms.double(450.5),
+                            Nybins = cms.int32(1350),
+                            ymin   = cms.double(0.5),
+                            ymax   = cms.double(1350.5),
+                            switch = cms.bool(False)
+                        ),
+                        PositionOfDigisSH = cms.PSet(
+                            Nxbins = cms.int32(1016),
+                            xmin   = cms.double(0.5),
+                            xmax   = cms.double(1016.5),
+                            Nybins = cms.int32(2),
+                            ymin   = cms.double(0.5),
+                            ymax   = cms.double(2.5),
+                            switch = cms.bool(True)
+                        ),
+                        PositionOfDigisPH = cms.PSet(
+                            Nxbins = cms.int32(960),
+                            xmin   = cms.double(0.5),
+                            xmax   = cms.double(960.5),
+                            Nybins = cms.int32(32),
+                            ymin   = cms.double(0.5),
+                            ymax   = cms.double(32.5),
+                            switch = cms.bool(True)
+                        ),
+                        EtaH = cms.PSet(
+                            Nbins  = cms.int32(45),
+                            xmin   = cms.double(-4.5),
+                            xmax   = cms.double(4.5),
+                            switch = cms.bool(True)
+                        ),
+                        DigiChargeH = cms.PSet(
+                            Nbins = cms.int32(50),
+                            xmin   = cms.double(-0.5),
+                            xmax   = cms.double(49.5),
+                            switch = cms.bool(True)
+                        ), 
+                        TotalNumberOfDigisPerLayerH = cms.PSet(
+                            Nbins = cms.int32(1000),
+                            xmin   = cms.double(-0.5),
+                            xmax   = cms.double(4999.5),
```

Comment:
think in the PU case we can have much higher occupancies, see e.g: https://tinyurl.com/y4thv7xg
![image](https://user-images.githubusercontent.com/5082376/94133675-4324f600-fe61-11ea-9f56-91d1d349176b.png)

Reply:
> In this first test and, upon completing all the code comments, I noticed that 2D plots DigiRPosVsZPos, DigiXPosVsYPos, OccupancyInRZ, OccupancyInXY seem to have moved from TrackerPhase2ITDigi/DigiMonitor/ to TrackerPhase2OTDigi/DigiMonitor/ and now have some renderization problems (except DigiXPosVsYPos), most likely due to a very large number of bins in both x and y axis.
> Could you please review those plots?

Yes. I think somehow I switched the 2D plots on or off in between to test something. Now all these global histograms are present for both IT & OT Digi-Monitor, but they have very fine binning. So, do you suggest to make less binning? Or, I was thinking to switch off the chargeXYmap 2D histogram. The binning is higher for that layer level histogram too.